test/dlc_tests/ecs/tensorflow/inference/test_ecs_tensorflow_inference.py

```diff
@@ -27,6 +27,29 @@ def test_ecs_tensorflow_inference_cpu(tensorflow_inference, ecs_container_instan
         ecs_utils.tear_down_ecs_inference_service(ecs_cluster_arn, service_name, task_family, revision)
 
 
+@pytest.mark.model("half_plus_two")
```

Comment:
nit: it's fairly clear from the name, but can you also add `@pytest.mark.integration("elastic_inference")`